osi_sensorview.proto

```diff
@@ -313,8 +313,10 @@ message CameraSensorView
 
     // Raw image data.
     //
-    // The raw image data in the memory layout and order specified by the
-    // camera sensor input configuration.
+    // The raw image data in the memory layout specified by the camera    
+    // sensor input configuration. The pixel order is specified in 
+    // CameraSensorViewConfiguration.pixel_order with the 
+    // default value PIXEL_ORDER_LEFT_RIGHT_TOP_BOTTOM.
```

Comment:
@RIFJo I just found another wrong enum in this comment. As soon as this is fixed we should probably add the label ReadyForCCB to this PR, right?

Reply:
I have fixed the codumentation. I think i am unable to add the ReadyForCCB label, someone else needs to do that.